vip-helpers/vip-utils.php

```diff
@@ -1292,7 +1292,8 @@ function wpcom_vip_should_load_plugins() {
 
 	$should_load_plugins = true;
 
-	if ( defined( 'WP_INSTALLING' ) && WP_INSTALLING ) {
+	// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+	if ( defined( 'WP_INSTALLING' ) && WP_INSTALLING && strtok( $_SERVER['REQUEST_URI'] ?? '', '?' ) !== '/wp-activate.php' ) {
```

Comment:
The condition becomes a bit harder to read. Should we break down + autocomment with variables a bit? 

Feel free to rename and reformat the changes.

```suggestion
	// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
	is_wp_activate_request = strtok( $_SERVER['REQUEST_URI'] ?? '', '?' ) === '/wp-activate.php';
	is_installing = defined( 'WP_INSTALLING' ) && WP_INSTALLING;
	if ( $is_installing && ! $is_wp_activate_request ) {
```

Reply:
I think `wp_installing()` might do the trick

Reply:
made the change, I think it's ok now.